src/Image.js

```diff
@@ -14,6 +15,11 @@ const Image = styled.div([], {
   props => props.css
 )
 
+Image.propTypes = {
+  width: PropTypes.string,
```

Comment:
Since styled-system accepts numbers, arrays, or strings, it's probably better to use the library's prop types like above